From 190a9fdb4bbe478b858d4511e5502c66369503aa Mon Sep 17 00:00:00 2001
From: Philip Balister <philip@balister.org>
Date: Thu, 4 Aug 2016 08:20:55 -0400
Subject: [PATCH] Use CMAKE_INSTALL_LIBDIR to set LIB_SUFFIX.

OpenEmbedded sets this variable for builds, so use it to figure out if
the build should install into the lib64 directories.

Signed-off-by: Philip Balister <philip@balister.org>
---
 cmake/Modules/GrPlatform.cmake | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/cmake/Modules/GrPlatform.cmake b/cmake/Modules/GrPlatform.cmake
index fbbea5f..00a53d0 100644
--- a/cmake/Modules/GrPlatform.cmake
+++ b/cmake/Modules/GrPlatform.cmake
@@ -51,4 +51,12 @@ endif()
 if(NOT DEFINED LIB_SUFFIX AND LIB64_CONVENTION AND CMAKE_SYSTEM_PROCESSOR MATCHES "64$")
     set(LIB_SUFFIX 64)
 endif()
+
+########################################################################
+# Detect /lib versus /lib64
+########################################################################
+if (CMAKE_INSTALL_LIBDIR MATCHES lib64)
+    set(LIB_SUFFIX 64)
+endif()
+
 set(LIB_SUFFIX ${LIB_SUFFIX} CACHE STRING "lib directory suffix")
-- 
2.5.5

